node/service/src/lib.rs

```diff
@@ -955,6 +967,19 @@ pub fn new_full<RuntimeApi, Executor>(
 		task_manager.spawn_essential_handle().spawn_blocking("babe", babe);
 	}
 
+	// Start BEEFY
+	task_manager.spawn_essential_handle().spawn_blocking(
```

Comment:
Should we do this only on rococo/westend? Otherwise if BEEFY crashes for some reason the whole node goes down.